TextToSpeech/impl/TTSManager.cpp

```diff
@@ -138,18 +162,27 @@ TTS_Error TTSManager::setConfiguration(Configuration &configuration) {
         }
         else {
             updated |= m_defaultConfiguration.setVoice(voices.front());
-            updated |= m_defaultConfiguration.setLanguage(configuration.language);
+            updated |= language_updated = m_defaultConfiguration.setLanguage(configuration.language);
```

Comment:
This makes it really hard to read.  I'd suggest to split it in two lines:

language_updated = m_defaultConfiguration.setLanguage(configuration.language);
updated |= language_updated;